cli-messaging/src/main/java/com/salesforce/messaging/EventKey.java

```diff
@@ -0,0 +1,59 @@
+package com.salesforce.messaging;
+import static com.salesforce.messaging.Message.*;
+
+public enum EventKey {
+	// MAKE SURE `messageKey` OF EVERY VALUE ADDED HERE HAS AN ENTRY IN 'messages/EventKeyTemplates.js'!
+	INFO_GENERAL_INTERNAL_LOG("info.generalInternalLog", 1, MessageType.INFO, MessageHandler.INTERNAL, true),
+	WARNING_INVALID_CAT_SKIPPED("warning.invalidCategorySkipped", 1, MessageType.WARNING, MessageHandler.UX, true),
+	WARNING_INVALID_RULESET_SKIPPED("warning.invalidRulesetSkipped", 1, MessageType.WARNING, MessageHandler.UX, true),
+	WARNING_XML_DROPPED("warning.xmlDropped", 1, MessageType.WARNING, MessageHandler.UX, true),
+	INFO_JAR_AND_XML_PROCESSED("info.jarAndXmlProcessed", 2, MessageType.INFO, MessageHandler.UX, true),
+	ERROR_INTERNAL_UNEXPECTED("error.internal.unexpectedError", 1, MessageType.ERROR, MessageHandler.INTERNAL, false),
+	ERROR_INTERNAL_MAIN_INVALID_ARGUMENT("error.internal.mainInvalidArgument", 1, MessageType.ERROR, MessageHandler.INTERNAL, false),
+	ERROR_INTERNAL_JSON_WRITE_FAILED("error.internal.jsonWriteFailed", 1, MessageType.ERROR, MessageHandler.INTERNAL, false),
+	ERROR_INTERNAL_CLASSPATH_DOES_NOT_EXIST("error.internal.classpathDoesNotExist", 1, MessageType.ERROR, MessageHandler.INTERNAL, false),
+	ERROR_INTERNAL_XML_MISSING_IN_CLASSPATH("error.internal.xmlMissingInClasspath", 1, MessageType.ERROR, MessageHandler.INTERNAL, false),
+	ERROR_EXTERNAL_JAR_NOT_READABLE("error.external.jarNotReadable", 1, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_DIR_NOT_READABLE("error.external.dirNotReadable", 1, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_DUPLICATE_XML_PATH("error.external.duplicateXmlPath", 3, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_MULTIPLE_RULE_DESC("error.external.multipleRuleDesc", 2, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_RECURSION_LIMIT("error.external.recursionLimitReached", 2, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_XML_NOT_READABLE("error.external.xmlNotReadable", 2, MessageType.ERROR, MessageHandler.UX, false),
+	ERROR_EXTERNAL_XML_NOT_PARSABLE("error.external.xmlNotParsable", 2, MessageType.ERROR, MessageHandler.UX, false),
+	WARNING_MULTIPLE_METHOD_TARGET_MATCHES("warning.multipleMethodTargetMatches", 3, MessageType.WARNING, MessageHandler.UX, true),
```

Comment:
I created both new messages as warnings. The "multiple matches found" is verbose-only, the "no matches found" is always.
Question: Might it be better to make the former `info` instead of `warning`? I could see it going either way.